runtime/lua/vim/filetype/util.lua

```diff
@@ -0,0 +1,49 @@
+local M = {}
```

Comment:
No `util.lua`, just put these in `filetype.lua` directly and they can be accessed as `vim.filetype.whatever`.